CONTRIBUTING.adoc

```diff
@@ -27,7 +63,18 @@ apm install
 
 === Prepare for business
 
-*todo*
+. Decide what you want to work on, and what should be the result
+. Have a GitHub account
+. Fork the project to your personal account
+. Check out your forked project for local development using git
+. Navigate to the directory using the command line interface
+. Link the package using `apm link -d`, and start atom in dev-mode using the command `atom -d .`
+. _Develop_ (this is where the magic happens)
+. Commit and push your code onto your forked project
+. Make a pull request to get the changes merged into the upstream project
```

Comment:
You might want to mention here that we use a pull request template so the contributor knows to expect it. Ask them to fill it out to the best of their abilities.